cinema/urls.py

```diff
@@ -15,6 +15,7 @@
 router.register("cinema_halls", CinemaHallViewSet)
 router.register("movies", MovieViewSet)
 router.register("movie_sessions", MovieSessionViewSet)
+router.register("orders", OrderViewSet)
 
 urlpatterns = [path("", include(router.urls))]
```

Comment:
In case `urlpatterns` consist of only router urls, you could assign `router.urls` to `urlpatterns` directly